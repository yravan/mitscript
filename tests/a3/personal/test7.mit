i = 0;
j = 0;
while (i <= 1) {
  print("in loop");
  i = i + 1;
  while (j < 1) {
    print("in inner loop");
    j = j + 1;
  }
}

while (5 + 8 < -2) {
  print("should not print");
}
print("done");