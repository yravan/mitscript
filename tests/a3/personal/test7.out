in loop
in inner loop
in loop
done
